app/[[...slug]]/page.tsx

```
@@ -0,0 +1,91 @@
+      <article className="prose lg:prose-xl max-w-none">
+        <header className="mb-8">
+          <div className="bg-red-500 text-white p-2 font-mono text-center">
+            Generated at: {new Date().toLocaleTimeString()}
+          </div>
+          <h1 className="text-4xl font-bold mb-2">{page.title}</h1>
+          <time className="text-gray-500 text-sm">
+            {new Date(page.updated_at).toLocaleDateString()}
+          </time>
+        </header>
+
+        <div dangerouslySetInnerHTML={{ __html: page.content ?? "" }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)